src/docs/product/accounts/membership.mdx

```diff
@@ -60,36 +61,69 @@ Member roles dictate access within an organization.
+
+### Team Admin
+
+- Admin access to teams of which they're members.
+- Can create new teams and Projects, as well as remove Projects and teams of which they're members.
+
+### Team Contributor
+
+- Can view and act on issues.
+- Can join and leave teams (depending on the rules set by the org owner/manager).
+- Can invite new members to join (pending approval from org owner/manager).
+- Can add Projects for team (depending on the rules set by the org owner/manager).
+
 ## Open Membership
 
-Open membership is enabled by default, which allows members to join, leave, and add other members to teams.
+Open membership is enabled by default. This allows users to join, leave, and add other members to teams.
 
 ## Restricting Access
 
-- Access to a Project is limited to the Team that owns the project. However, with open membership,
-  members can join and add members to teams.
-- To further restrict access to a Project, you can control access to Teams by making them accessible
-  only through invitation by a Manager, Owner, or Admin.
+- Project access is limited to the team that owns the Project, but org members can get access to the Project by joining the team that owns it (depending on the rules set by the org owner/manager).
+- To limit access to a Project, you can limit access to the team that owns that Project by making it invite-only. Users will only be able to join from an invitation by a Manager, Owner, or Admin.
 
-To restrict Team access, go to Organization Settings and flip the “Open Membership” toggle.
+To restrict team access, go to the **Settings page> General Settings** and flip the “Open Membership” toggle.
 
 ![](membership-toggle.png)
 
 ## Transferring a Project
 
-To transfer a project go to the target project's Project Settings >> General
-Settings >> Transfer Project. Enter the email of the organization's owner you
-wish to transfer it to, and they'll receive an email to approve the transfer.
-When the project is transferred to the new organization, it's recommended to
-add it to a team for better visibility.
+To transfer a Project to a new owner, go to the target Project's **Project Settings > General Settings > Transfer Project**. Enter the email address of the person you’d like to transfer it to. They'll receive an email to approve the transfer.
+
+When transferring a Project to a new organization, we recommend also adding a new team to the Project for better visibility.
 
 ## Removing an Organization
 
-It is possible to remove an organization and all associated
-organization data completely. Doing so will _not_ remove user accounts, but will remove
-their membership in the organization.
+If you’re an org owner, you can remove an entire organization and all its associated data completely. Doing so will *not* remove user accounts, but it will remove user membership in that organization.
```

Comment:
```
If you’re an org owner, you can remove an entire organization and all its associated data completely. 

A user can be a member in several Sentry organizations at the same time. As such, deleting the organization will *not* delete user accounts, but it will remove user's membership in that organization.
```